packages/loader/container-loader/api-report/container-loader.api.md

```diff
@@ -26,7 +26,7 @@ import { ITelemetryBaseLogger } from '@fluidframework/core-interfaces';
 import { ITelemetryLoggerExt } from '@fluidframework/telemetry-utils';
 import { IUrlResolver } from '@fluidframework/driver-definitions';
 
-// @alpha (undocumented)
+// @public (undocumented)
 export enum ConnectionState {
```

Comment:
Missed public export: exposed by FluidContainer

Reply:
there are multiple things in the code base with this name. are you sure this is needed? i would have expected build errors if it was